docs/concepts/use-cases.md

```diff
@@ -131,3 +131,10 @@ Network Security (IPS/IDS) Integration Example: [ProtectWise](https://xsoar.pan.
 - Scan assets for a specific vulnerability.
 
 Vulnerability Management Integration Example: [Tenable.io](https://xsoar.pan.dev/docs/reference/integrations/tenableio)
+
+## IAM (Identity and Access Management)
```

Comment:
Should this be: ILM (Identify Lifecycle Management)?

Reply:
The use case is "Identity and Access Management". We just gave a different more generic name to our pack.